cogboard-webapp/src/hooks/index.js

```diff
@@ -5,11 +5,13 @@ import { splitPropsGroupName } from '../utils/components';
 
 export const useToggle = (initialState = false) => {
   const [isOpened, setOpened] = useState(initialState);
+  const [expanded, setExpanded] = useState(false);
 
   const handleOpen = () => setOpened(true);
   const handleClose = () => setOpened(false);
+  const handleToggle = () => setExpanded(!expanded);
```

Comment:
`setOpened(!isOpened)` You don't need to declare another state holding the same values.